src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -62,7 +62,7 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   private PersistentStorageAgent persistentStorageAgent;
 
   /** The in-memory list of Users. */
-  private List<User> users;
+  private static List<User> users;
```

Comment:
I made it static because it was giving me a compiling error with that sort of thing. Do you know of the solution to the this?

Reply:
I think you probably want to get rid of the `static` on both this and the `getUsersList()` function. Then to access the function, you'd do this:

```
UserStore.getInstance().getUsersList();
```

That way the function doesn't have to be static, because you're going to the instance returned from `getInstance()`.

Reply:
Got it. Thank you!